app/styles/_set-home-page.less

```diff
@@ -0,0 +1,10 @@
+.modal-body-indent {
```

Comment:
These classes are very specific to this dialog and should be scoped to just the home page modal.